MMA.md

```diff
@@ -0,0 +1,250 @@
+ MMA (Mobile Marketing Automation)
+================================
+
+Mozilla wants to engage with users more. MMA is the project for this purpose. When a user performs a certain UI action (or set of UI actions), she will see a prompt and have a chance to interact with it. For example, if a user uses Firefox 10 times a week, but Firefox is not her default browser, we'll prompt the user the next time when she launches our app, and guide her to set us as default browser.
+
+Starting in Firefox 8.0 on iOS, Mozilla is using a third party framework called "Leanplum" in order to achieve this. Leanplum is a San Francisco company, founded in 2012. We put their SDK in our codebase via Carthage.
+
+The SDK is documented at https://www.leanplum.com/docs/ios
+
+There are three major component in Leanplum SDK.
+1. Events : Events are fired when users perform certain actions.
+2. User Attributes: User Attributes are set on a per-user basis, and inform us about an aspect of the user.
+3. Deep Links: Actions that users can perform to interact with the Message.
+4. Messages:  User Interaction points that we want to engage with users that help them use Firefox better.
+
+An Event or a series of Events plus some User Attributes may trigger a Message, and when the user acts on the Message, a Deep Link may be processed.
+
+
+Data collection
```

Comment:
Nit: This looks like a dangling fragment, maybe it should be formatted as a header?